relayer/src/relayer.rs

```diff
@@ -679,42 +683,51 @@ impl RelayerImpl {
 
         let l_subscriptions = subscriptions.read().unwrap();
 
+        let senders = if forward_all {
+            l_subscriptions.iter().collect::<Vec<(
```

Comment:
should be able to get by w/o collect here

Reply:
senders can be an iter instead of collected to vec, iter'd over then thrown away